packages/app-extensions/src/notifier/modules/modalComponents/StyledModalContent.js

```diff
@@ -48,6 +64,39 @@ const StyledModalContent = styled.div.attrs({
         outline: none;
       }
     }
+
+    ${StyledTitleWrapper}, ${StyledAdvancedSearchButtonWrapper} {
```

Comment:
works like a charm but i think in this case we should prefer class names / selector over styled-component. imo this would be more transparent and less coupled. Import gives it away ('../../../')